core/src/main/scala/org/apache/spark/deploy/SparkSubmit.scala

```diff
@@ -283,8 +284,17 @@ object SparkSubmit extends CommandLineUtils {
       } else {
         Nil
       }
+
+    // Create the IvySettings, either load from file or build defaults
+    val ivySettings = if (Option(args.ivySettingsFile).isDefined) {
```

Comment:
true, that is kind of dumb I guess.. I just realized that since this is just a conf anyway and not a command-line arg, I don't need `args.ivySettingsFile` and I could just do 
```
val ivySettings = args.sparkProperties.get("spark.jars.ivySettings").map { ivySettingsFile => ...
```
Look ok to you?